extensions/typescript-language-features/package.json

```diff
@@ -480,6 +480,18 @@
             "tags": [
               "experimental"
             ]
+          },
+          "typescript.maximumHoverLength": {
```

Comment:
For new settings like these, let's start using a single setting instead of duplicating it for javascript and typescript. IO suggest calling it: `js/ts.hover.maximumLength`

If you set the scope to `"resource"`, then it can be configured per language as well using language specific settings

Reply:
You'll need to get the `js/ts` configuration section: `const jsTsConfig = vscode.workspace.getConfiguration('js/ts', document)`.